packages/core/bootstrap/src/lib/server.ts

```diff
@@ -55,23 +51,21 @@ export const initHandler =
       setupMetricsServer(name)
     }
 
-    initExpressMiddleware(app)
-
     const executeWithMiddleware = await withMiddleware(execute, context, middleware)
 
-    app.post(baseUrl, (req, res) => {
-      if (!req.is(CONTENT_TYPE_APPLICATION_JSON)) {
-        return res
-          .status(HTTP_ERROR_UNSUPPORTED_MEDIA_TYPE)
-          .send(HTTP_ERROR_UNSUPPORTED_MEDIA_TYPE_MESSAGE)
-      }
-      req.body.data = {
-        ...(req.body.data || {}),
+    app.post(baseUrl, async (req, res) => {
+      ;(req.body as AdapterRequest).data = {
+        ...((req.body as AdapterRequest).data || {}),
```

Comment:
I feel like the existing code read better. Any particular reason to changing it?

```typescript
req.body.data = {
        ...(req.body.data || {}),
```

Reply:
there is a type problem when it is done the other way, it is necessary to specify that it is an AdapterRequest between the parentheses otherwise it fails. Could you give it a try?🙏

Reply:
Can you give this a try? :smile: 

```typescript
    app.post<{
      Body: AdapterRequest
    }>(baseUrl, async (req, res) => {
      req.body.data = {
        ...(req.body.data || {}),
        ...toObjectWithNumbers(req.query),
      }
      return executeSync(
        req.body,
        executeWithMiddleware,
        context,
        (status, result) => {
          res.code(status).send(result)
        },
      )
    })
```